### ISSUE TITLE
Incorrect path in error when failing to load properties.

### ISSUE THREAD

**Issue by kokarn** [rahatarmanahmed/get-package-info#21]
I'm looking into `electron-prebuilt` and stumbled upon an error.

When failing to load properties the error that comes from `resolve` doesn't really make much sense.

If I run it from `/home/kokarn/my-app/` it reports
```
Cannot find module 'package.json' from '/home/kokarn'(Properties not found yet: dependencies.electron-prebuilt,devDependencies.electron-prebuilt)
```

The problem with that error is both that the path is incorrect and the actual error message is incorrect. 
It does actually find the file but it's just missing some properties.

Maybe and updated error message would be nice?
A separation between when we fail to load `package.json` and when we fail to find properties would be very helpful.

I'm happy to submit a PR if we can agree on what to output.

**Comment by rahatarmanahmed** [comment]
Yup, I've been waiting on https://github.com/substack/node-resolve/pull/81 to get merged but it doesn't look like that will happen any time soon. We'll need to switch to another library, so you can try to make a PR for #2. I also found https://www.npmjs.com/package/find-package-json which seems like a way better fit, but it uses generators. That might be ok since I think electron-packager only supports node v4+.

**Comment by rahatarmanahmed** [comment]
Oh nvm, I don't think find-package-json will work, since it doesn't give you the directory the package came from. We need that contextual info.

**Comment by kokarn** [comment]
Because of the scope of this package I'd probably look at findup instead. Seems to be a quicker implementation as well.

**Comment by rahatarmanahmed** [comment]
Fixed in 0827710bd6de41a012485b0a29a4299b9a63505d
